rclcpp_examples/src/parameters/ros2param.cpp

```diff
@@ -44,7 +55,8 @@ parse_args(int argc, char ** argv, std::string & remote_node, param_operation_t
     return rclcpp::parameter::ParameterVariant();
   }
   remote_node = name.substr(0, slash);
-  std::string variable = name.substr(slash + 1, name.size() - slash - 1);
+  variable = name.substr(slash + 1, name.size() - slash - 1);
```

Comment:
The declaration should stay in this line: see [Declare variables in the narrowest scope possible](https://github.com/ros2/ros2/wiki/Developer-Guide#programming-conventions)